proto/sifnode/oracle/v1/types.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package sifnode.oracle.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/Sifchain/sifnode/x/oracle/types";
+
+// GenesisState - all clp state that must be provided at genesis
+//TODO: Add parameters to Genesis state ,such as minimum liquidity required to create a pool
+message GenesisState {
+  repeated string address_whitelist = 1;
+  string          admin_address     = 2;
+}
+
+// Claim contrains an arbitrary claim with arbitrary content made by a given validator
+message Claim {
+  string id                = 1;
+  string validator_address = 2;
```

Comment:
I don't see why we can't use a custom gogo proto type here as well for this instead of just using a string.